fiboa_cli/datasets/digifarm.py

```diff
@@ -0,0 +1,146 @@
+# that the user knows they need to get the data from the API.
+# SOURCES = "https://gist.githubusercontent.com/cholmes/de82be680b6f9a14ea9e61cf5cbd9c91/raw/2b30dd30a6ed409c23f5181272e20ef3acd4808e/digifarm-norway-sample.json"
+SOURCES = None
+
+DATA_ACCESS = """
+Data must be obtained from the Digifarm API, see https://api-docs.digifarm.io/. Just save any
+API output to a GeoJSON file (with .json as the file extension) and use that as the input file.
+If you want some DigiFarm data, anyone can use:
+https://api.digifarm.io/v1/delineated-fields?token=a0731a8c-5259-4c68-af3a-7ad4f6d53faa&bbox=11.13,60.72,11.21,60.76
+Just save the result as something like 'digifarm-sample.json' and then use -i with the file name to convert it.
+"""
+
+# A filter function for the layer in the file(s) to read.
+# Set to None if the file contains only one layer or all layers should be read.
+# Function signature:
+#   func(layer: str, path: str) -> bool
+LAYER_FILTER = None
+
+# Unique identifier for the collection
+ID = "digifarm"
+# Geonames for the data (e.g. Country, Region, Source, Year)
+SHORT_NAME = "DigiFarm Field Boundaries"
+# Title of the collection
+TITLE = "Field boundaries created by DigiFarm Automatic Field Delineation Model"
+# Description of the collection. Can be multiline and include CommonMark.
+DESCRIPTION = """These field boundaries are created by DigiFarm, using an  state-of-the-art deep neural network model for Field Delineation
+from super-resolved satellite imagery. The results are available through an API, covering over 200 million hectares across 30+ countries. 
+The data is provided through the DigiFarm API at https://api-docs.digifarm.io/, as GeoJSON. For more information see https://digifarm.io/products/field-boundaries
+"""
+
+# A list of providers that contributed to the data.
+# This should be an array of Provider Objects:
+# https://github.com/radiantearth/stac-spec/blob/master/collection-spec/collection-spec.md#provider-object
+PROVIDERS = [
+    {
+        "name": "DigiFarm",
+        "url": "https://digifarm.io",
+        "roles": ["producer", "licensor"]
+    }
+]
+
+# Attribution (e.g. copyright or citation statement as requested by provider).
+# The attribution is usually shown on the map, in the lower right corner.
+# Can be None if not applicable
+ATTRIBUTION = "© 2024 digifarm.io"
+
+# License of the data, either
+# 1. a SPDX license identifier (including "dl-de/by-2-0" / "dl-de/zero-2-0"), or
+# LICENSE = "CC-BY-4.0"
+# 2. a STAC Link Object with relation type "license"
+LICENSE = {"title": "Proprietary", "href": "https://digifarm.io/legal/tc", "type": "text/html", "rel": "license"}
+
+# Map original column names to fiboa property names
+# You also need to list any column that you may have added in the MIGRATION function (see below).
+COLUMNS = {
+    "area_ha": "area",
+    "id": "id",
+    "geometry": "geometry",
+    "area": "area_sq_m",
```

Comment:
We use hectares, not m²

Reply:
Their area field is in sq meters. So what I'm trying to do here is just use their hectares as area, and then use their area as just a custom field for digifarm called area_sq_m.

But I then got some data that didn't have area_ha, so I think the best approach is likely to do area: area, and then do the column transform.

Reply:
Ah, gotcha. Sounds good. The value of having m² in addition to ha seems pretty limited to me...

Reply:
Yeah, I was trying to stay 'faithful' to theirs:

```
      "properties": {
        "id": "10149_no_202012",
        "area": 100095.056069513,
        "area_acres": "24.734",
        "area_ha": "10.010"
      },
```

I'll ask to see if they feel it is really important to have all 3.

Reply:
area_ha as a string? And the rounding is also a bit weird... :-o

Reply:
To clean things up I'll convert from area (m²) to hecatares and ignore the other fields.